components/admin/data/layers/table/component.js

```diff
@@ -89,101 +68,39 @@ class LayersTable extends PureComponent {
       filters: {
         ...filters,
         name: value
```

Comment:
I know you didn't touch this, but let's not send the `name` field if the value is empty. This way we avoid unexpected behaviors from the API.

Reply:
@andresgnlez I tried to follow that approach but it's not possible. Consider for instance the case where the user performs a search such as "food" and then deletes the text in order to see all results. The state would keep having `name: food` if we only add the name property with a condition such as `...(value !== '' && { name: value })`